tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -1830,7 +1835,12 @@ def test_OnDiskDataset_preprocess_yaml_content_windows():
         with open(yaml_file, "w") as f:
             f.write(yaml_content)
 
-        preprocessed_metadata_path = gb.preprocess_ondisk_dataset(test_dir)
+        with pytest.warns(
+            DGLWarning,
+            match="Edge feature is stored, but edge IDs are not saved.",
+        ):
+            preprocessed_metadata_path = gb.preprocess_ondisk_dataset(test_dir)
```

Comment:
how about either specifying `include_original_edge_id=True` or remove the edge feature instead of capturing the warning? As edge feature are usually added, I prefer `include_original_edge_id=True`. From my understanding, we just need to capture warning for the tests that disable the `include_xx`.

Reply:
DONE